### PR TITLE
[CORE-2084] WorkOrders Info | Add distance

### PR DESCRIPTION
**Pull request by tiagobastos**

### CODE REVIEW

**Review by WouterSioen — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

File: src/09-workorders/work-orders.apib [hunk 1]
```diff
@@ -37,6 +37,7 @@ Get details for a single work order.
                 + type: `event` (string)
                 + id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string)
             + location (Address)
+            + distance (Distance, optional)
```
Comment (dietrichm) [inline src/09-workorders/work-orders.apib]: B:
```suggestion
            + distance (Distance)
```

**Review by dietrichm — DISMISSED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Comment by JordyDeGroeve** [comment]
Q: so what do we return if there is no distance given?

**Comment by mrtus** [comment]
@JordyDeGroeve @dietrichm maybe make it `nullable`?

**Comment by WouterSioen** [comment]
I also think this should be nullable

**Comment by tiagobastos** [comment]
In the proposal the distance is optional in the `.draft` and optional & nullable in the `.update`.
This being the `.info` is ok like this, right? 
It's consistent with all the others.

**Comment by WouterSioen** [comment]
If it can be `null` or a `Distance` (since we support not passing it through draft and making it `null` through update), we should probably also reflect that in the `info` endpoint? It's not like we're going to make up some random data if there is no distance, right?

**Review by JordyDeGroeve — DISMISSED** [review]
Also, as @tiago and @WouterSioen noticed, we should not allow floats to be sent as the value. This should be updated too.

**Review by WouterSioen — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Pull request merged**